src/compas_rhino/conversions/surfaces.py

```diff
@@ -295,61 +71,33 @@ def surface_to_compas_quadmesh(surface, nu, nv=None, weld=False, facefilter=None
     weld: bool, optional
         Weld the vertices of the mesh.
         Default is False.
-    facefilter: callable, optional
-        A filter for selection which Brep faces to include.
-        If provided, the filter should return True or False per face.
-        A very simple filter that includes all faces is ``def facefilter(face): return True``.
-        Default parameter value is None in which case all faces are included.
-    cls: :class:`compas.geometry.Mesh`, optional
+    cls: :class:`compas.datastructures.Mesh`, optional
```

Comment:
what is this for?

Reply:
ah is this so you can get e.g. a `Polyhedron` instead?

Reply:
no, if you want a subclass of mesh instead of the default